src/bcs/MFEMVectorDivDirichletBC.C

```diff
@@ -0,0 +1,16 @@
+#include "MFEMVectorDivDirichletBC.h"
+
+registerMooseObject("PlatypusApp", MFEMVectorDivDirichletBC);
+
+MFEMVectorDivDirichletBC::MFEMVectorDivDirichletBC(const InputParameters & parameters)
+  : MFEMVectorDirichletBCBase(parameters)
+{
+}
+
+void
+MFEMVectorDivDirichletBC::ApplyBC(mfem::GridFunction & gridfunc, mfem::Mesh * mesh_)
+{
+  mfem::Array<int> ess_bdrs(mesh_->bdr_attributes.Max());
+  ess_bdrs = GetMarkers(*mesh_);
+  gridfunc.ProjectBdrCoefficientNormal(*_vec_coef->getVectorCoefficient(), ess_bdrs);
+}
```

Comment:
It's worth us adding a test for this BC before including it - it should be covered by MFEM Example 4, by analogy with our inclusion of the tangential vector Dirichlet BC in MFEM Example 3.

As it's not used in the linear elastic test, I'd be tempted to remove `MFEMVectorDivDirichletBC` from this PR and add it alongside a Platypus version of MFEM Example 4.